product/reports/170_Configuration Management - Containers/120_Projects by Number of Containers.yaml

```diff
@@ -0,0 +1,44 @@
+---
+title: Projects by Number of Containers
+rpt_group: Custom
+rpt_type: Custom
+priority:
```

Comment:
@nimrodshn if this is what other predefined reports have (empty) then OK :+1: for me.